src/domains/Find/common/components/NewPasswordInput.jsx

```
@@ -19,7 +19,7 @@ const NewPasswordInput = ({ register, errors, watch }) => {
           />
           <div className='flex items-start'>
             {errors.newPassword && (
```

ESLint

[error] 'errors.newPassword' is missing in props validation

(react/prop-types)